e2e/define_var/WORKSPACE

```diff
@@ -1,3 +1,5 @@
+workspace(name = "examples_define_var")
```

Comment:
why bother naming it?

Reply:
consistency is the only reason. don't feel strongly about it either way.